examples/restaurant_demo/restaurant.py

```diff
@@ -0,0 +1,341 @@
+import pydp as dp
+from pydp.algorithms.laplacian import BoundedSum, Count
+
+import math
+import statistics as s
+import pandas as pd
+from collections import defaultdict
+
+# An hour when visitors start entering the restaurant (900 represents 9:00 AM)
+OPENING_HOUR = 900
+
+# An hour when visitors stop entering the restaurant (2000 represents 20:00 PM)
+CLOSING_HOUR = 2000
+
+# Range of valid work hours when a visitor can enter the restaurant.
+VALID_HOURS = list(range(OPENING_HOUR, CLOSING_HOUR + 1))
+
+# For how many hours visitors can enter the restaurant
+NUM_OF_WORK_HOURS = CLOSING_HOUR - OPENING_HOUR + 1
+
+# Number of visit days contributed by a single visitor will be limited to 3. All exceeding visits will be discarded.
+COUNT_MAX_CONTRIBUTED_DAYS = 3
+
+# Number of visit days contributed by a single visitor will be limited to 4. All exceeding visits will be discarded.
+SUM_MAX_CONTRIBUTED_DAYS = 4
+
+# Minimum amount of money we expect a visitor to spend on a single visit.
+MIN_EUROS_SPENT = 0
+
+# Maximum amount of money we expect a visitor to spend on a single visit.
+MAX_EUROS_SPENT_1 = 50
+MAX_EUROS_SPENT_2 = 65
+
+LN_3 = math.log(3)
+
+
+class RestaurantStatistics:
+    def __init__(self, hours_filename, days_filename, epsilon=LN_3):
+
+        # Store the name of the csvs for daily and weekly restaurant data
```

Comment:
CSV _files_ or .csv _files_ - to be more explicit

```diff
-         # Store the name of the csvs for daily and weekly restaurant data
+        # Store the filenames (in csv) with daily and weekly restaurant data
```

Reply:
Done